x-pack/plugins/uptime/server/lib/adapters/telemetry/kibana_telemetry_adapter.ts

```diff
@@ -66,6 +66,13 @@ export class KibanaTelemetryAdapter {
             },
             overview_page: { type: 'long' },
             settings_page: { type: 'long' },
+            fleet_monitor_name_stats: {
+              avg_length: { type: 'float' },
+              max_length: { type: 'long' },
+              min_length: { type: 'long' },
+            },
+            fleet_monitor_frequency: { type: 'array', items: { type: 'long' } },
+            fleet_no_of_unique_monitors: { type: 'long' },
```

Comment:
@dominiqueclarke do you mind documenting the fields by adding the `_meta.description` like:

```
{ type: 'float', _meta: { description: 'This field represents...' } }
```